Flutter/lib/NameScreen.dart

```diff
@@ -1,30 +1,39 @@
-import 'package:flutter/material.dart';
-import 'package:google_fonts/google_fonts.dart';
-import 'CreateSettle.dart';
-import 'JoinSettle.dart';
-import 'app_localizations.dart';
-
-class NameScreen extends StatelessWidget {
-  final _formKey = GlobalKey<FormState>();
+import 'import_all.dart';
 
+class NameScreen extends StatefulWidget {
+  final DarkThemeProvider themeChange;
   final bool newSession;
   final context;
-  NameScreen(this.newSession, this.context);
+  NameScreen(this.newSession, this.context, this.themeChange);
```

Comment:
Again, makes more sense if DarkThemeProvider is managed by AppTheme

Reply:
Resolved in commit `app-style 43887b4`